roboteam_ai/src/control/controlGoToPos/ControlGoToPosLuTh.h

```diff
@@ -149,7 +149,8 @@ class ControlGoToPosLuTh {
 
           struct CustomCompare {
             bool operator()(NumRobotPtr lhs, NumRobotPtr rhs) {
-                if (lhs->collisions < rhs->collisions) return false;
+                if (lhs->collisions > rhs->collisions) return true;
```

Comment:
does this mean lefthandside and righthandside? Maybe a little vague...

Reply:
this is how they did it on the interwebs :)

Reply:
but what does it mean?